src/requestCertificate.scss

```diff
@@ -5,3 +5,8 @@
 .storage-row > * {
   margin-right: 1.5rem;
 }
+
+.dns-helper-text {
```

Comment:
This is because the textarea and the input doesn't quite work the same and it gives you a little extra gap, compared to the [example that uses input](https://www.patternfly.org/v4/documentation/react/components/form#examples), right?
Would be good to file this as a Patternfly issue, so we don't have to reimplement this for any helper text for textareas.

Reply:
Mentioned here: https://github.com/skobyda/cockpit-certificates/pull/16#discussion_r448797087